addons/studio-invite-button/userscript.js

```diff
@@ -0,0 +1,57 @@
+export default async function ({ addon, console, msg }) {
+  // invites a user to the studio
+  function inviteToStudio(user) {
+    return fetch(
+      `https://scratch.mit.edu/site-api/users/curators-in/${
+        location.pathname.split("/")[2]
+      }/invite_curator/?usernames=${user}`,
+      {
+        headers: {
+          "x-csrftoken": addon.auth.csrfToken,
+          "x-requested-with": "XMLHttpRequest",
+          referer: "https://scratch.mit.edu",
+        },
+        body: null,
+        method: "PUT",
+      }
+    );
+  }
+  addon.tab.waitForElement("#description").then(async (description) => {
+    // checks if the user is a manager and the URL is correct
+    if (description.classList.contains("editable")) {
+      for (;;) {
```

Comment:
use while?

Reply:
`for(;;)` is more readable imo, if it's the general practice to use while in scratchaddons ill switch though